<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadCourseClass</class>
 <widget class="QMainWindow" name="uploadCourseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uploadCourse</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Upload  New Course:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Course ID:&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Course Title:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Credit Hours:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Syllabus / Description: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Instructor Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCourseId">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CourseTitle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit_CourseCreditHours">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_InstructorName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_CourseDesc">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>701</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadEditCourseBtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>540</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>600</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
